src/project1.js

```diff
@@ -1,15 +1,18 @@
 function processData(response) {
-    const project = response.project;
-    let output = "";
+  const project = response.project;
 
-    document.getElementById("pageTitle").innerHTML = "PROJECTS";
-    document.getElementById("title").innerHTML = "Analytics Platform";
-        for (let i = 0; i < project.length; i++) {
-            output += project[i].info[0] + "<p>";
-            output += project[i].info[1] + "<p>";
-            output += project[i].info[2];
-        }
-        document.getElementById("projects").innerHTML = output;
-    }
+  document.getElementById("pageTitle").innerHTML = "PROJECTS";
+  document.getElementById("project1-title").innerHTML = "Analytics Platform";
+  for (let i = 0; i < project.length; i++) {
+    const myList = document.createElement("p");
+    let listArray = document.getElementById("project1-info");
+    listArray.appendChild(myList).insertAdjacentHTML(
+      "afterbegin",
+      `${project[i].info[0]} <p>
```

Comment:
If you find yourself hard-coding array indices like this, give a good think on perhaps iterating over the array instead